clap-v3-utils/src/offline.rs

```diff
@@ -53,7 +53,7 @@ fn signer_arg<'a>() -> Arg<'a> {
         .value_parser(value_parser!(PubkeySignature))
         .requires(BLOCKHASH_ARG.name)
         .multiple_occurrences(true)
-        .multiple_values(true)
+        .multiple_values(false)
```

Comment:
i think we can just delete the property here? single value is default iirc

Reply:
It's not clear from the documentation nor the code that it defaults to single value, so I wrote up a test to make sure, and you're right, it defaults to one value.

Since it's not clear just from reading the code / docs, I'd prefer to leave it as is for clarity, but I don't feel too strongly about it.

Reply:
sg :shipit: